scripts/EN/Harry

```diff
@@ -3314,14 +3314,14 @@ CleanScreen(); // 0x4000: 00000064
 LoadBackground(0x154); // 0x4004: 00001069
 DrawBackground(); // 0x400a: 00000065
 PlaySecondSoundFx(0xc); // 0x400e: 00001073
-ShowText((1289)"After leaving the kitchen I head right and place my hand on the first door that I come to. I notice the words ¥"Director's Office¥" out of the corner of my eye as I open the door. From inside the room a strong stench overflows outward and causes me to feel dizzy."); // 0x4014: 0000100c
+ShowText((1289)"After leaving the kitchen I head right and place my hand on the first door that I come to. W1I notice the words¥nW2¥"Director's Office¥" out of the corner of my eye as I open the door. W1From inside the room a strong stench overflows outward and causes me to feel dizzy."); // 0x4014: 0000100c
 HandleInput(); // 0x401a: 00000010
 PlaySecondSoundFx(0x1f); // 0x401e: 00001073
 CleanScreen(); // 0x4024: 00000064
 LoadBackground(0x1d0); // 0x4028: 00001069
 DrawBackground(); // 0x402e: 00000065
 LoadSecondEffect(0xd, 0x1, 0x0); // 0x4032: 0000306e
-ShowText((1290)"Someone has ransacked the room. Behind the desk that probably belongs to the director several bottles have been broken leaving red and blue liquid spilled all over the floor."); // 0x403c: 0000100c
+ShowText((1290)"Someone has ransacked the room. ¥nW2Behind the desk that probably belongs to the director several glass vials have been broken leaving red and blue liquid spilled all over the floor."); // 0x403c: 0000100c
```

Comment:
Changed bottle to "glass vial" to match the PS1 version of the game.

(Bottle is more accurate, but maybe they used vial here to avoid confusion with pet/plastic bottle?